index.html

```diff
@@ -1055,7 +1055,9 @@
             the user agent; for example it MAY show the user a dialog and allow
             the user to select an available device (<em>granting
             permission</em>), or cancel the selection (<em>denying
-            permission</em>).
+            permission</em>). The user agent is encouraged to render user
+            friendly names of available devices, using their locale and text
+            direction when they are known.
```

Comment:
The Presentation API text around was more specific, suggesting that the locale and text direction originate from the remote playback device (not necessarily the preferences of the user agent displaying the name).

https://w3c.github.io/presentation-api/#selecting-a-presentation-display

Reply:
OK, I've slightly updated the text to clarify the message (for once, we cannot encourage the user agent to render "user friendly names" without clarifying that we're not expecting the user agent to invent user friendly names but rather to use the ones it has discovered).